main.lua

```diff
@@ -41,13 +42,13 @@ for _, Folder in ipairs(cFile:GetFolderContents(WorldEditPath)) do repeat
 	dofolder(Path)
 until true end
 
-PLUGIN = nil
+
+
 
 
 function Initialize(a_Plugin)
-	PLUGIN = a_Plugin
-	PLUGIN:SetName(g_PluginInfo.Name)
-	PLUGIN:SetVersion(g_PluginInfo.Version)

```

Comment:
Should cWorld also be an exception? Technically they can be stored right? Since we do have the cRoot:GetWorld method.